mysite/profile/tests.py

```diff
@@ -235,18 +240,26 @@ class MockFetchPersonDataFromOhloh(object):
 # Mockup of stump's contribution list as given by ohloh, stripped down and
 # slightly tweaked for the purposes of testing.
 stumps_ohloh_results = mock.Mock()
-stumps_ohloh_results.return_value = ([
-    {u'contributor_name': u'stump', u'analysis_id': u'1145788',
-     u'man_months': u'11', u'primary_language_nice_name': u'C',
-     u'contributor_id': u'2008814186590608'},
-    {u'contributor_name': u'John Stumpo', u'analysis_id': u'1031175',
-     u'man_months': u'12', u'primary_language_nice_name': u'Python',
-     u'contributor_id': u'110891760646528'}
-], WebResponse())
+stumps_ohloh_results.return_value = (
+    [
+        {u'contributor_name': u'stump',
+         u'analysis_id': u'1145788',
+         u'man_months': u'11',
+         u'primary_language_nice_name': u'C',
+         u'contributor_id': u'2008814186590608'},
+        {u'contributor_name': u'John Stumpo',
```

Comment:
This { looks almost like it ran astray, might be better if they had their own lines.